src/components/Earn.jsx

```diff
@@ -294,6 +388,13 @@ export default function Earn() {
             !serviceInfo?.makerRunning &&
             !isWaitingMakerStart &&
             !isWaitingMakerStop && <p className="text-secondary mb-4">{t('earn.market_explainer')}</p>}
+          {serviceInfo?.makerRunning && serviceInfo?.offers && serviceInfo?.nickname && (
+            <>
```

Comment:
I'd show one of our default placeholders here since we can already anticipate that an offer will load (since `makerRunning` is true).

Reply:
Hmm.. the Earn page definitely needs some overhaul regarding loading, etc.
Will add a loader for the "active offer" box, but inclined to do some general refactoring in a follow-up PR.